src/test/java/edu/cornell/kfs/rass/batch/xml/fixture/RassXmlAwardEntryFixture.java

```diff
@@ -473,9 +465,13 @@ public Boolean getNullSafeFinalFinancialReportRequired() {
         return defaultToFalseIfNull(finalFinancialReportRequired);
     }
 
-    private java.sql.Date buildSqlDateFromDateTime(DateTime dateTime) {
-        if (dateTime != null) {
-            return new java.sql.Date(dateTime.getMillis());
+    // Then section of this method must match the processing in RassValueConverterBase.cleanDataValue.
+    // Naming of the two methods pertains to the functionality each is perforning for the test case versus
+    // the actual data file processing but they fundamentally are performing the same logic.
+    private Date buildSqlDateFromLocalDate(LocalDate localDateToConvert) {
+        if (localDateToConvert != null) {
+            LocalDate localDate = LocalDate.of(localDateToConvert.getYear(), localDateToConvert.getMonthValue(), localDateToConvert.getDayOfMonth());
+            return Date.valueOf(localDate);
```

Comment:
If you do make updates to the `RassValueConverterBase.cleanDateValue()` method, then please update this area to match the logic accordingly.